<launch>
    <arg name="base_foot_print" default="car/car_base_link" />
    <arg name="map_frame" default="3dmap" />
    <arg name="serial_addr" default="/dev/ttyS1" />
    <arg name="localization" default="true" />
    <arg name="rviz" default="true" />

    <!-- <include file="/home/hpf/ARX5/ros/src/arx5/arx5_moveit/launch/arx5_moveit.launch" />  -->
    

    <node pkg="arx5_control" type="arx_server.py" name="arx_server" /> 

   <group if="$(arg rviz)">
   
       <node name="rviz" pkg="rviz" type="rviz" args="-d $(find navigation)/rviz/navi.rviz" required="true" />
   </group> 


    <include if="$(arg localization)"  file="$(find hdl_localization)/launch/hdl_localization.launch"/>

    <arg name="use_sim" default="false"/>
    <arg name="use_rviz" default="false"/>
    <arg name="drag" default="false"/>
<!-- use_sim="$(arg use_sim) " use_rviz="$(arg use_rviz)" drag="$(arg drag)"  -->
    

  
    <node pkg="navigation" type="navi_service" name="navi_service" output="screen" />
        <param name="base_foot_print" value="$(arg base_foot_print)" />
        <param name="map_frame" value="$(arg map_frame)" />

    <node pkg="robot_teleop" type="keyboard_nav.py" name="teleop_twist_keyboard_nav_node" /> 
    
    <node pkg="navigation" type="navigation" name="navigation"  />
        <!-- <param name="base_foot_print" value="$(arg base_foot_print)" output="screen"/>
        <param name="map_frame" value="$(arg map_frame)" />
        <param name="serial_addr" value="$(arg serial_addr)" /> -->

</launch>
